lib/protocol/payloads/deltas/remote_conflicts.ts

```diff
@@ -31,18 +31,14 @@ export class DeltaRemoteConflicts extends PayloadsDelta {
         payload.uuid!,
         PayloadSource.DecryptedTransient
       );
-      if (!decrypted) {
+      if (!decrypted && !payload.deleted) {
         /** Decrypted should only be missing in case of deleted payload */
-        if (!payload.deleted) {
-          throw 'Unable to find decrypted counterpart for data conflict.';
-        }
-        results.push(payload);
```

Comment:
Just want to make sure this line wasn't overlooked when simplifying the `if` statement? I assume the pushing here is handled instead by passing this payload to the conflict delta.

Reply:
Exactly. This seemed like a responsibility for the `ConflictDeta` class and not its callers. The test would have covered that oversight as well, however your comment made me think of an way to make said test tighter.